chatminer/chatparsers.py

```diff
@@ -68,15 +68,15 @@ def __init__(self, filepath):
     def parse_file(self):
         self._logger.info("Starting reading raw messages...")
         self._read_raw_messages_from_file()
-        self._logger.info("Finished reading %i raw messages.", len(self._raw_messages))
+        self._logger.info(f"Finished reading {len(self._raw_messages)} raw messages.")
 
         self._logger.info("Starting parsing raw messages...")
         self._parse_raw_messages()
         self._logger.info("Finished parsing raw messages.")
 
     @abstractmethod
     def _read_raw_messages_from_file(self):
-        return
+        ...
```

Comment:
If this fails the codecov report, how about using `pass` instead?
According to a short googling session, there doesn't seem to be a straight forward fix for codecov.

Reply:
The only reason it is failing is because when I had it like this in the first commit:
```py
@abstractmethod
    def _read_raw_messages_from_file(self): ...
```
CodeCov knew that it was a stub and so didn't take it into account for the code coverage. In the next commit I changed it to what it currently is to satisfy black but that makes it so CodeCov thinks it needs to be covered which isn't true. Basically it is only complaining because I did it the "correct" way the first time.

Reply:
I see, it's on the tool then. Let's ignore it, it should be fine compared with main anyways